example/example-buffers.py

```diff
@@ -30,3 +30,16 @@
     for j in range(m4.cols()):
         print(m4[i, j], end = ' ')
     print()
+
+from example import ConstructorStats
+cstats = ConstructorStats.get(Matrix)
```

Comment:
Indeed, but rather than have `gc.collect()` calls all over the place, the first thing `cstats.alive()` does is a garbage collector invocation.